app/(product)/public-cloud/(product)/aws-roles/[licencePlate]/[role]/page.tsx

```diff
@@ -96,41 +96,72 @@ export default function ProductAWSRoles() {
     refetch();
   }, [searchTerm, refetch]);
 
+  const {
+    data: roles,
+    isLoading: isRolesFetching,
+    error: fetchingRolesError,
+  } = useQuery<any, Error>({
+    queryKey: ['licencePlate', licencePlate],
+    queryFn: () => getRolesNames(licencePlate),
+    enabled: !!licencePlate,
+  });
+
+  if (roles && roles.length === 0) {
+    return (
+      <div className="w-full">
+        Looks like role groups haven&apos;t been create for this product, please, reach out Public Cloud Platform
```

Comment:
I believe there is a typo?

Reply:
eslint error `112:37  error  `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`  react/no-unescaped-entities`

Reply:
I meant `been create` -> `been created`

Reply:
yes, typo, edited, thanks